<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LSMainWindowClass</class>
 <widget class="QMainWindow" name="LSMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LumiSuite</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="MoudleNameLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>34</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(210, 210, 210);</string>
          </property>
          <property name="text">
           <string>模组名称</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ModuleNamelineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>34</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>34</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="MoudleSelectLayout">
        <property name="horizontalSpacing">
         <number>1</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QPushButton" name="ImportBtn">
          <property name="styleSheet">
           <string notr="true">QPushButton{color: rgb(255, 255, 255);border-top-color: rgb(180, 180, 180);
border-right-color: transparent;
border-left-color: transparent;
border-bottom-color:  transparent;}
QPushButton{background-color: rgb(87,94,136);color:white;}
QPushButton:hover{background-color: rgb(117,149,209);color:white;}
QPushButton:pressed{background-color: rgb(71, 175, 255);color:white;}</string>
          </property>
          <property name="text">
           <string>导入</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="CreateBtn">
          <property name="styleSheet">
           <string notr="true">QPushButton{color: rgb(255, 255, 255);border-top-color: rgb(180, 180, 180);
border-right-color: transparent;
border-left-color: transparent;
border-bottom-color:  transparent;}
QPushButton{background-color: rgb(87,94,136);color:white;}
QPushButton:hover{background-color: rgb(117,149,209);color:white;}
QPushButton:pressed{background-color: rgb(71, 175, 255);color:white;}</string>
          </property>
          <property name="text">
           <string>Create</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="DeleteBtn">
          <property name="styleSheet">
           <string notr="true">QPushButton{color: rgb(255, 255, 255);border-top-color: rgb(180, 180, 180);
border-right-color: transparent;
border-left-color: transparent;
border-bottom-color:  transparent;}
QPushButton{background-color: rgb(87,94,136);color:white;}
QPushButton:hover{background-color: rgb(117,149,209);color:white;}
QPushButton:pressed{background-color: rgb(71, 175, 255);color:white;}</string>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="ExportBtn">
          <property name="styleSheet">
           <string notr="true">QPushButton{color: rgb(255, 255, 255);border-top-color: rgb(180, 180, 180);
border-right-color: transparent;
border-left-color: transparent;
border-bottom-color:  transparent;}
QPushButton{background-color: rgb(87,94,136);color:white;}
QPushButton:hover{background-color: rgb(117,149,209);color:white;}
QPushButton:pressed{background-color: rgb(71, 175, 255);color:white;}</string>
          </property>
          <property name="text">
           <string>导出</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="CopyBtn">
          <property name="styleSheet">
           <string notr="true">QPushButton{color: rgb(255, 255, 255);border-top-color: rgb(180, 180, 180);
border-right-color: transparent;
border-left-color: transparent;
border-bottom-color:  transparent;}
QPushButton{background-color: rgb(87,94,136);color:white;}
QPushButton:hover{background-color: rgb(117,149,209);color:white;}
QPushButton:pressed{background-color: rgb(71, 175, 255);color:white;}</string>
          </property>
          <property name="text">
           <string>复制</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="SaveBtn">
          <property name="styleSheet">
           <string notr="true">QPushButton{color: rgb(255, 255, 255);border-top-color: rgb(180, 180, 180);
border-right-color: transparent;
border-left-color: transparent;
border-bottom-color:  transparent;}
QPushButton{background-color: rgb(87,94,136);color:white;}
QPushButton:hover{background-color: rgb(117,149,209);color:white;}
QPushButton:pressed{background-color: rgb(71, 175, 255);color:white;}</string>
          </property>
          <property name="text">
           <string>保存</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="MoudleFileList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>34</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(210, 210, 210);</string>
        </property>
        <property name="text">
         <string>模组文件列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidgetList">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>309</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>380</width>
          <height>16777215</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string>模组名称</string>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="informationBar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="MoudleFileList_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>34</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>34</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(210, 210, 210);</string>
        </property>
        <property name="text">
         <string>图形参数列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="tableView">
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(210, 210, 210);</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,8,0,1,3,1,3,1,3,0,0">
         <item>
          <widget class="QCheckBox" name="CASDevice">
           <property name="text">
            <string>CAS设备</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>279</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="UpBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{color: rgb(255, 255, 255);border-top-color: rgb(180, 180, 180);
border-right-color: transparent;
border-left-color: transparent;
border-bottom-color:  transparent;}
QPushButton{background-color: rgb(87,94,136);color:white;}
QPushButton:hover{background-color: rgb(117,149,209);color:white;}
QPushButton:pressed{background-color: rgb(71, 175, 255);color:white;}</string>
           </property>
           <property name="text">
            <string>上移</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>113</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="DownBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{color: rgb(255, 255, 255);border-top-color: rgb(180, 180, 180);
border-right-color: transparent;
border-left-color: transparent;
border-bottom-color:  transparent;}
QPushButton{background-color: rgb(87,94,136);color:white;}
QPushButton:hover{background-color: rgb(117,149,209);color:white;}
QPushButton:pressed{background-color: rgb(71, 175, 255);color:white;}</string>
           </property>
           <property name="text">
            <string>下移</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>112</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="AddBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{color: rgb(255, 255, 255);border-top-color: rgb(180, 180, 180);
border-right-color: transparent;
border-left-color: transparent;
border-bottom-color:  transparent;}
QPushButton{background-color: rgb(87,94,136);color:white;}
QPushButton:hover{background-color: rgb(117,149,209);color:white;}
QPushButton:pressed{background-color: rgb(71, 175, 255);color:white;}</string>
           </property>
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>113</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="DelBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{color: rgb(255, 255, 255);border-top-color: rgb(180, 180, 180);
border-right-color: transparent;
border-left-color: transparent;
border-bottom-color:  transparent;}
QPushButton{background-color: rgb(87,94,136);color:white;}
QPushButton:hover{background-color: rgb(117,149,209);color:white;}
QPushButton:pressed{background-color: rgb(71, 175, 255);color:white;}</string>
           </property>
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="SaveParmsBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{color: rgb(255, 255, 255);border-top-color: rgb(180, 180, 180);
border-right-color: transparent;
border-left-color: transparent;
border-bottom-color:  transparent;}
QPushButton{background-color: rgb(87,94,136);color:white;}
QPushButton:hover{background-color: rgb(117,149,209);color:white;}
QPushButton:pressed{background-color: rgb(71, 175, 255);color:white;}</string>
           </property>
           <property name="text">
            <string>保存</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="ConnectDevice"/>
   <addaction name="DisconnectDevice"/>
   <addaction name="separator"/>
   <addaction name="ImageProcessing"/>
   <addaction name="moduleObjectFile"/>
  </widget>
  <action name="moduleObjectFile">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Resources/edit.png</normaloff>:/Resources/edit.png</iconset>
   </property>
   <property name="text">
    <string>模组文件</string>
   </property>
   <property name="toolTip">
    <string>模组文件编辑</string>
   </property>
  </action>
  <action name="ImageProcessing">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Resources/imageprocessing.png</normaloff>:/Resources/imageprocessing.png</iconset>
   </property>
   <property name="text">
    <string>图像处理</string>
   </property>
   <property name="toolTip">
    <string>图像处理</string>
   </property>
  </action>
  <action name="ConnectDevice">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Resources/LC_1.png</normaloff>:/Resources/LC_1.png</iconset>
   </property>
   <property name="text">
    <string>连接设备</string>
   </property>
   <property name="toolTip">
    <string>设备初始化</string>
   </property>
  </action>
  <action name="DisconnectDevice">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/Resources/LC_1.png</normaloff>:/Resources/LC_1.png</iconset>
   </property>
   <property name="text">
    <string>断开设备</string>
   </property>
   <property name="toolTip">
    <string>断开连接</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
